librz/type/type.c

```diff
@@ -787,17 +787,21 @@ RZ_API ut64 rz_type_db_get_bitsize(const RzTypeDB *typedb, RZ_NONNULL RzType *ty
 	return 0;
 }
 
-static char *type_as_string(const RzTypeDB *typedb, RZ_NONNULL const RzType *type, bool prev_ptr, bool is_first) {
-	rz_return_val_if_fail(typedb && type, NULL);
+struct HelperBufs {
+	RzStrBuf *arraybuf;
+	RzStrBuf *ptrbuf;
+};
```

Comment:
Please use typedef and place this on top of the file, so we always have in a .c or .h file

1. macros
2. types
2. definitions
3. functions

Reply:
There is no need to use typedef in this case, I used `struct` on purpose, to show the true essence of this parameter - it's just a pair of buffers. And it's located before 3 functions it's used in. It is unnecessary everywhere else and shouldn't be used outside those.

Reply:
I agree no typedefs necessary when the thing is local to a file